src/components/primitives/Bullet.tsx

```diff
@@ -17,4 +17,4 @@ Bullet.defaultProps = {
   size: 16,
 };
 
-export default Bullet;
+export { Bullet };

```

Comment:
❓ Should we do `export const Bullet` to keep the styling consistent?

Reply:
Sure, make sense!